src/TwigRenderer.php

```diff
@@ -65,25 +68,34 @@ public function __construct(TwigEnvironment $template = null, string $suffix = '
 
     /**
      * Create a default Twig environment
+     *
+     * @param FilesystemLoader $loader
+     *
+     * @return Environment
      */
-    private function createTemplate(TwigFilesystem $loader) : TwigEnvironment
+    private function createTemplate(FilesystemLoader $loader) : Environment
     {
-        return new TwigEnvironment($loader);
+        return new Environment($loader);
     }
 
     /**
      * Get the default loader for template
      */
-    private function getDefaultLoader() : TwigFilesystem
+    private function getDefaultLoader() : FilesystemLoader
     {
-        return new TwigFilesystem();
+        return new FilesystemLoader();
     }
 
     /**
      * Render
      *
+     * @param string       $name
      * @param array|object $params
-     * @throws \Zend\Expressive\Template\Exception\InvalidArgumentException for non-array, non-object parameters.
+     *
+     * @return string
+     * @throws LoaderError
+     * @throws RuntimeError
+     * @throws SyntaxError
```

Comment:
Please revert the `@param` and `@return` annotations, and _expand_ the `@throws` annotations to indicate what scenarios will result in those specific error types.